helmchart/sdp/values.yaml

```diff
@@ -202,6 +216,8 @@ sdp:
       DATABASE_URL: #required
       DISTRIBUTION_SEED: #required
       CHANNEL_ACCOUNT_ENCRYPTION_PASSPHRASE: #required
+      KAFKA_SASL_USERNAME: #optional, depends on value of KAFKA_SECURITY_PROTOCOL
```

Comment:
Isn't username meant to be in the confihgOptions while just the password lives in the secrets?

Reply:
Discussed with Marcelo through DM. I'm keeping username as secret for the sake of consistency because we're doing the same thing for Twilio and AWS.